data-structures-stacks/count-values.js

```diff
@@ -0,0 +1,20 @@
+// export default function countValues(stack) {
+//   const popped = 0;
+//   for (let i = 0; i < 4; i++) {
+//     const top = stack.peek();
+//     popped.push(top);
+//     stack.pop();
+//   }
+//   return popped.length;
+// }
+
+export default function countValues(stack) {
+  let popped = 0;
+  for (let i = 0; i < 5; i++) {
```

Comment:
Why only 5? You're lucky the tests don't use stacks with more than 5 elements in them. :-)

I think you saw that `while` is the best solution, which you now get to add to your list of things to learn more about. :-)

Reply:
Wel, because 5 worked 😂 

Ha, exactly.